### ISSUE TITLE
Games not loading when controller connected

### ISSUE THREAD

**Issue by PeteRepeat77** [D4rkC00d3r/xctvplayer#2]
Xbox series X controller on Google tv. When connected games don't load, just a short fade of screen. Disconnecting controller let's game start after choosing continue anyway at popup screen. Reconnect controller while take is loading and all works ok.

**Comment by D4rkC00d3r** [comment]
Thanks I can reproduce this in some cases and I am looking into a cause.

**Comment by D4rkC00d3r** [comment]
I think I have a solution for this but will require more testing.

![gamepad](https://user-images.githubusercontent.com/17799879/147486540-fc9ebd64-098d-4fe4-9d5b-457608a267c6.png)

**Comment by anouarghali** [comment]
I am having the same issue, tried the disconnect controller work around and it seems to do the job. I am not on CCwGTV, I am on a Philips android tv with Oreo

**Comment by PeteRepeat77** [comment]
If you wish for a beta tester to trial this I am happy to. I have challenges connecting my Xbox series X controller to the ccwgtv so to disconnect and then reconnect can be a little challenging at times.

**Comment by anouarghali** [comment]
I would love to test this, where can find the latest packaged APK with the controller fix?
________________________________

**Comment by kjbaker-uk** [comment]
I have not released one yet as I just haven't got it reliable yet, for now turning on the joypad after starting the game still seems the best option for the moment. When I have a build ready I will post it just in this issue thread.

**Comment by kjbaker-uk** [comment]
Just a update to say that I have not yet found a stable solution. The work-around is still the best option.

**Comment by PeteRepeat77** [comment]
Thanks for the update.

**Comment by PeteRepeat77** [comment]
I guess this has reached the end of development so this will never be resolved?

**Comment by kjbaker-uk** [comment]
Hi Pete, sorry mate this is tricky as it is Microsoft end that is preventing this from working correctly. I have tried to take control of the gamepad which I can do and connection to my client app is established but Xcloud does not see it.  Not sure what to do about it. Also how much time do I put into this?

**Comment by PeteRepeat77** [comment]
I tend to use my shield these days, but when I was using this app (as I have Ccwgtv scattered everywhere) I found it could take quite a few times to get the xbox controller reconnected to the Ccwgtv which sometimes involved repairing a few times. Thus why it was a bit of a pain.

**Comment by kjbaker-uk** [comment]
Agreed mate. I am now either playing on my Series X or Xcloud on my Steamdeck.
